src/app/shared/components/disk-offering/disk-offering-dialog/disk-offering-dialog.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="table-wrapper"
+      *ngIf="diskOfferings?.length">
+  <table class="mat-table">
+    <thead>
+    <tr class="mat-header-row">
+      <ng-container *ngFor="let column of columns">
+        <th class="mat-header-cell"> {{ tableId.toUpperCase() + '.' + column?.toUpperCase() | translate }}</th>
```

Comment:
please, try to get rid of concatenation
If it's not possible, get rid of toUpperCase calls